app/views/layouts/application_layout.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class ApplicationLayout < ApplicationView
+	include Phlex::Rails::Layout
+
+	def template(&block)
+		doctype
+
+		html do
+			head do
+				title { "You're awesome" }
+				meta name: "viewport", content: "width=device-width,initial-scale=1"
+				csp_meta_tag
+				csrf_meta_tags
+				stylesheet_link_tag "application", data_turbo_track: "reload"
+				javascript_importmap_tags
+			end
+
+			body do
+				main(&block)
+			end
+		end
+	end
+end
```

Comment:
I'm excited about this, @Jordybastien :raised_hands: 

Indentation is a bit crazy though :laughing: